client/extensions/zoninator/state/zones/test/reducer.js

```diff
@@ -157,6 +160,52 @@ describe( 'reducer', () => {
 			} );
 		} );
 
+		it( 'should initiate zones array for a site ID after the first zone is added', () => {
```

Comment:
OK, maybe "initiate" should be "create" then? Or perhaps you were going for "initialize"?